egs/librispeech/ASR/conformer_ctc/conformer.py

```diff
@@ -15,18 +29,18 @@
 class Conformer(Transformer):
     """
     Args:
-        num_features (int): Number of input features
-        num_classes (int): Number of output classes
-        subsampling_factor (int): subsampling factor of encoder (the convolution layers before transformers)
-        d_model (int): attention dimension
-        nhead (int): number of head
-        dim_feedforward (int): feedforward dimention
-        num_encoder_layers (int): number of encoder layers
-        num_decoder_layers (int): number of decoder layers
-        dropout (float): dropout rate
-        cnn_module_kernel (int): Kernel size of convolution module
-        normalize_before (bool): whether to use layer_norm before the first block.
-        vgg_frontend (bool): whether to use vgg frontend.
+      num_features (int): Number of input features
```

Comment:
Did you reformat this file by hand or using `black`?

I just ran black on this file and nothing is changed.

(I am using black==21.6b0)

Reply:
I reverted the reformatting of conformer.py. It seems some conflicts between black and flake8, so ignored the E203 checking for that line.
![image](https://user-images.githubusercontent.com/11765074/130378069-a0b11802-9310-48ae-b0ad-640b288d49cc.png)